packages/unified-latex-to-pretext/libs/pre-conversion-subs/macro-subs.ts

```diff
@@ -29,13 +43,36 @@ function factory(
     };
 }
 
+function createMessage(node: Ast.Macro, replacement: string): VFileMessage {
```

Comment:
Define one version of create message in a separate file and imported it in the two places its used.